apps/condo/domains/organization/integrations/sbbol/sync/syncUser.js

```diff
@@ -59,6 +61,16 @@ const cleanEmailForAlreadyExistingUserWithGivenEmail = async ({ email, userIdToE
     }
 }
 
+const registerIdentity = async ({ context, user, userInfo }) => {
+    await registerUserExternalIdentity(context.context, {
+        ...dvSenderFields,
+        user: { id: user.id },
+        identityId: userInfo.importId,
+        identityType: SBBOL_IDP_TYPE,
```

Comment:
it should be as argument

Reply:
This function are located in sbbol/sync/syncUser - that means all methods correspond to this topic (SBBOL). In syncUser method we are importing only sbbol data into the platform. So, i believe identityType should be hardcoded and no need to rename syncUser to syncSbbolUser